### PR TITLE
Update mingw-w64

### PR DESCRIPTION
**Pull request by lazka**

### CODE REVIEW

File: mingw-w64-winstorecompat-git/PKGBUILD [hunk 1]
```diff
@@ -5,7 +5,7 @@ pkgbase=mingw-w64-${_realname}-git
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
 pkgver=9.0.0.6246.ae63cde27
```
Comment (Biswa96) [inline mingw-w64-winstorecompat-git/PKGBUILD:6]: pkgver is same.

**Comment by MehdiChinoune** [comment]
Why not using [pkgver()](https://wiki.archlinux.org/title/creating_packages#pkgver())?

**Comment by Biswa96** [comment]
All the mingw-w64 pkgbuild files are using `pkgver()` function.

**Comment by jeremyd2019** [comment]
Note that commit 0e0eed623 *fixes* the intended behavior that `time_t` on 32-bit UCRT defaults to 64 instead of 32 bits.  This is potentially an ABI breakage for clang32, the only 32-bit UCRT currently in existence here.  /cc @revelator since you seem to be using clang32 as well.

**Comment by lazka** [comment]
Not sure if that warrants rebuilds since clang32... I'd be fine with leaving things as is for now.

**Comment by jeremyd2019** [comment]
Sounds good to me.  I can rebuild things if/when there's a failure, or maybe grep include files for `time_t` to try to find packages where `time_t` is in their external interface.

**Comment by Biswa96** [comment]
😈 

1. Delete `/clang32` in repo
2. rebuild everything in it.

**Pull request merged**